HeterogeneousCore/AlpakaUtilities/BuildFile.xml

```diff
@@ -0,0 +1,13 @@
+<use name="rootcore"/>
+<use name="boost"/>
+<use name="eigen"/>
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SoATemplate"/>
```

Comment:
As far as I can tell, none of these dependencies are used. Or is there further code in the pipeline that would bring in some of these dependencies?